tasks/libs/package/size.py

```diff
@@ -31,6 +34,26 @@
     },
 }
 
+PACKAGE_SIZE_TEMPLATE = {
+    'amd64': {
+        'datadog-agent': {'deb': 140000000},
+        'datadog-iot-agent': {'deb': 10000000},
+        'datadog-dogstatsd': {'deb': 10000000},
+        'datadog-heroku-agent': {'deb': 70000000},
+    },
```

Comment:
What are the numbers ? A size limit ?

Reply:
Yes it's a size increase limit. I kept the same values from the existing check. I think we can update the comparison method to be more efficient and/or fine-grained. This will most probably come in a further update.

Reply:
Could you add a comment to explain that ? It's not very intuitive (to me) just looking at the code